src/commands/misc/role.ts

```diff
@@ -0,0 +1,211 @@
+import {
+  ChatInputCommandInteraction,
+  GuildMember,
+  MessageReaction,
+  PartialMessageReaction,
+  PartialUser,
+  Role,
+  SlashCommandBuilder,
+  User,
+} from "discord.js";
+
+import { discordClient } from "../../clients/discord";
+
+const ROLES_CHANNEL = "741479910886866944";
+
+const THEY_THEM = "741479573337800706";
+const HE_HIM = "741479366319538226";
+const SHE_HER = "741479514902757416";
+const LISTENER = "466622497991688202";
+const NO_ALERTS = "512522219574919179";
+const LFG = "754473984661258391";
+
+const EMOJI_TO_ROLE = new Map([
+  ["🇹", THEY_THEM],
+  ["♀️", SHE_HER],
+  ["♂️", HE_HIM],
+  ["👂", LISTENER],
+  ["🚫", NO_ALERTS],
+  ["✅", LFG],
+]);
+
+export const data = new SlashCommandBuilder()
+  .setName("role")
+  .setDescription("Self-serve certain roles on the server")
+  .addSubcommand((subcommand) =>
+    subcommand
+      .setName("pronouns")
+      .setDescription("Please indicate your preferred pronouns")
+      .addStringOption((option) =>
+        option
+          .setName("pronoun")
+          .setDescription("Toggle a preferred pronoun. You can come back to set multiple")
+          .addChoices(
+            { name: "they/them", value: THEY_THEM },
```

Comment:
just wanted to note this -- on some of the blaseball servers i am on folks get into the xe/xir or he/she/they/xir type structures. imo this is something where so long as we are nonbinary friendly with they/them we can wait until there is a request for more substantial pronouns. or we can eventually get a full pronoun-bot like the [pronounsdb](https://github.com/NurMarvin/pronoundb-bot) api stuff later on if we really need more complex pronoun-ing.

Reply:
I agree we can ignore neo-pronouns for now and probably for a long time